workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/hooks/useKPI.ts

```diff
@@ -0,0 +1,196 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import useSWR from 'swr';
+
+import {useMarketplaceContext} from '../../../context/MarketplaceContext';
+import SearchBuilder from '../../../core/SearchBuilder';
+import {
+	PARTNERSHIP_TYPE,
+	ProductCategories,
+	ProductType,
+} from '../../../enums/Product';
+import HeadlessCommerceAdminCatalog from '../../../services/rest/HeadlessCommerceAdminCatalog';
+
+const baseSearchBuilder = new SearchBuilder()
+	.group('OPEN')
+	.lambdaContains('specificationValues', '2025 Q')
+	.or()
+	.lambdaContains('specificationValues', '2024 Q')
+	.or()
+	.lambdaContains('specificationValues', '2023 Q')
+	.group('CLOSE');
+
+const projectUsingMarketplaceAppsFilter = new URLSearchParams({
+	fields: 'totalCount',
+	pageSize: '-1',
+});
+
+const partnershipIntegrationFilter = new URLSearchParams({
+	fields: 'totalCount,productSpecifications',
+	filter: new SearchBuilder()
+		.lambdaContains(
+			'specificationValues',
+			PARTNERSHIP_TYPE.TECHNOLOGY_PARTNERSHIP
+		)
+		.build(),
+	nestedFields: 'productSpecifications',
+	pageSize: '-1',
+});
+
+const supportingQuartelyReleaseFilter = new URLSearchParams({
+	fields: 'totalCount,productSpecifications',
+	filter: baseSearchBuilder.clone().build(),
+	nestedFields: 'productSpecifications',
+	pageSize: '-1',
+});
+
+const connectorQuartelyReleaseFilter = new URLSearchParams({
+	fields: 'totalCount,productSpecifications',
+	filter: baseSearchBuilder
+		.clone()
+		.and()
+		.lambda('categoryNames', ProductCategories.PAYMENT_METHODS)
+		.build(),
+	nestedFields: 'productSpecifications',
+	pageSize: '-1',
+});
+
+const lowCodeConfigurationsPublishedFilter = new URLSearchParams({
+	fields: 'totalCount,productSpecifications,categories',
+	filter: new SearchBuilder()
+		.lambdaContains(
```

Comment:
I guess that the exact match is better on this case

```suggestion
		.lambda(
```